api/server/logs.go

```diff
@@ -0,0 +1,42 @@
+package server
```

Comment:
I moved this file in the 'server' package to avoid a cyclic dependency issue between 'log' and 'server' packages importing each other.